server/public/styles.css

```diff
@@ -1,3 +1,6 @@
-h1 {
-  margin: 1rem 0;
+header {
+  margin: .75rem 0;
+}
+h4 {
+  text-align: right;
```

Comment:
I believe there are helper classes for stuff like this in Bootstrap

https://getbootstrap.com/docs/4.4/utilities/spacing/

https://getbootstrap.com/docs/4.4/utilities/text/

Reply:
I found the classes and used those instead